README.md

```diff
@@ -1,21 +1,319 @@
-# FeatureFirstHasuraConsolePoc
+# Feature-First Hasura Console API POC
 
-<a alt="Nx logo" href="https://nx.dev" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-logo.png" width="45"></a>
 
-✨ **This workspace has been generated by [Nx, a Smart, fast and extensible build system.](https://nx.dev)** ✨
+- [Context](#context)
+- [The problem](#the-problem)
+- [How the problem is tackled right now](#how-the-problem-is-tackled-right-now)
+- [What's the Platform team goal?](#whats-the-platform-team-goal)
+- [What to expect from this POC](#what-to-expect-from-this-poc)
+- [Basic use cases](#basic-use-cases)
+  - [🙋 As a developer, I want to **show a feature only if it's enabled**](#-as-a-developer-i-want-to-show-a-feature-only-if-its-enabled)
+  - [🙋 As a developer, I want to **refresh some data related to the Hasura Plan**](#-as-a-developer-i-want-to-refresh-some-data-related-to-the-hasura-plan)
+  - [🙋 As a developer, I want to be sure **my component is not rendered until the Hasura Plan data is available** because I do not want to manage the loading state](#-as-a-developer-i-want-to-be-sure-my-component-is-not-rendered-until-the-hasura-plan-data-is-available-because-i-do-not-want-to-manage-the-loading-state)
+  - [🙋 As a developer, I want to test my component in all its versions in **Storybook**](#-as-a-developer-i-want-to-test-my-component-in-all-its-versions-in-storybook)
+- [Advanced use cases](#advanced-use-cases)
+  - [🙋 As a developer, I want to **add one feature** to the catalogue of features managed by `useIsFeatureEnabled` (Neon, for instance)](#-as-a-developer-i-want-to-add-one-feature-to-the-catalogue-of-features-managed-by-useisfeatureenabled-neon-for-instance)
+  - [🙋 As a developer, I want to **add one more async source of Hasura Plan info**](#-as-a-developer-i-want-to-add-one-more-async-source-of-hasura-plan-info)
+  - [🙋 As a developer, I want to **add one more Console type**](#-as-a-developer-i-want-to-add-one-more-console-type)
+  - [🙋 As a developer, I want to manage the **dynamic route** for my feature](#-as-a-developer-i-want-to-manage-the-dynamic-route-for-my-feature)
+- [FAQ](#faq)
+- [How can I play with the demo?](#how-can-i-play-with-the-demo)
 
-## Development server
 
-Run `nx serve feature-first-hasura-console-poc` for a dev server. Navigate to http://localhost:4200/. The app will automatically reload if you change any of the source files.
 
-## Understand this workspace
 
-Run `nx graph` to see a diagram of the dependencies of the projects.
+<details id="context">
+  <summary><b>Context</b></summary>
+
+1. The Hasura Console can be loaded in six different "types", and two different "modes" (12 cases to manage). This is messy represented by the `window.__env` object the Hasura finds when loads.
```

Comment:
**typo:** Missing word *the Hasura **Console** finds when loads*.

Reply:
Done 😊